docs/src/main/asciidoc/qute.adoc

```diff
@@ -26,45 +26,48 @@ Clone the Git repository: `git clone {quickstarts-clone-url}`, or download an {q
 
 The solution is located in the `qute-quickstart` link:{quickstarts-tree-url}/qute-quickstart[directory].
 
-== Hello World with Jakarta REST
-
-If you want to use Qute in your Jakarta REST application, you need to add an extension first:
+[[hello-qute-web]]
+== Hello World with Qute Web
 
-* either `quarkus-rest-qute` if you are using Quarkus REST (formerly RESTEasy Reactive):
-+
+If you want to use Qute in your Quarkus Web application, add the Qute Web extension first:
 [source,xml,role="primary asciidoc-tabs-target-sync-cli asciidoc-tabs-target-sync-maven"]
 .pom.xml
 ----
 <dependency>
-    <groupId>io.quarkus</groupId>
-    <artifactId>quarkus-rest-qute</artifactId>
+    <groupId>io.quarkiverse.qute.web</groupId>
+    <artifactId>quarkus-qute-web</artifactId>
 </dependency>
 ----
 +
 [source,gradle,role="secondary asciidoc-tabs-target-sync-gradle"]
 .build.gradle
 ----
-implementation("io.quarkus:quarkus-rest-qute")
+implementation("io.quarkiverse.qute.web:quarkus-qute-web")
 ----
 
-* or `quarkus-resteasy-qute` if you are using RESTEasy Classic:
-+
-[source,xml,role="primary asciidoc-tabs-target-sync-cli asciidoc-tabs-target-sync-maven"]
-.pom.xml
-----
-<dependency>
-    <groupId>io.quarkus</groupId>
-    <artifactId>quarkus-resteasy-qute</artifactId>
-</dependency>
-----
-+
-[source,gradle,role="secondary asciidoc-tabs-target-sync-gradle"]
-.build.gradle
+All files located in the `src/main/resources/templates` directory and its subdirectories are registered as templates. Templates are validated during startup and watched for changes in the development mode.
+
+Now, let's start with a Hello World html template:
+
+.src/main/resources/templates/pub/hello.html
+[source]
 ----
-implementation("io.quarkus:quarkus-resteasy-qute")
+<h1>Hello {http:param('name', 'Quarkus')}!</h1> <1>
 ----
+<1> `{http:param('name', 'Quarkus')}` is an expression that is evaluated when the template is rendered (Quarkus is the default value).
 
-We'll start with a very simple template:
+NOTE: Templates located in the `pub` directory are served via HTTP. Automatically, no controllers needed. For example, the template src/main/resource/templates/pub/foo.html will be served from the paths /foo and /foo.html by default.
+
+If your application is running, you can open your browser and hit:  http://localhost:8080/hello?name=Martin
+
+For more information about Qute Web, see the https://docs.quarkiverse.io/quarkus-qute-web/dev/index.html[Qute Web guide].
+
+[[hello-qute-rest]]
+== Hello World with Jakarta REST
```

Comment:
I don't get it? Why would it go first? In both case you are serving templates?

The qute-web basic approach is just simpler until you need something more specific with Quarkus REST

Reply:
Because the Jakarta REST is IMHO a more simple and a more common use case.

You are trying to push your web story very hard in a guide that has nothing to do with your Quarkus Web guide.

For me the basic use case of using Qute with REST should come first.

(Also please don't mark conversations as resolved when they aren't as it makes it very hard to track where feedback was given)

Reply:
agreeing with gsmet here that in this guide the "fuller" web app is a bit secondary and qute with rest fits and then web app comes next